WordPress/WordPressUITests/Flows/EditorFlow.swift

```diff
@@ -7,6 +7,10 @@ class EditorFlow {
         }
     }
 
+    static func goToMySiteScreen() -> MySiteScreen {
```

Comment:
I do think it's a tad confusing that you changed the name from `gotoMySiteScreen` to `goToMySiteScreen` (see difference in capitalization in the first "T") but otherwise looks good!